pyrasterframes/src/main/python/setup.py

```diff
@@ -72,6 +69,59 @@ def run(self):
                 print(traceback.format_exc())
 
 
+class PweaveDocs(distutils.cmd.Command):
```

Comment:
Remove this from this PR. Covered by https://github.com/locationtech/rasterframes/pull/143